### DISCUSSION TITLE
Dynamic array of characters

### DISCUSSION THREAD

**Discussion by neu5** [Annoraaq/grid-engine#55] [Q&A] [answered]
Hey, @Annoraaq. As you know I'm building a game with my friend using your phaser plugin.
I was wondering what you would suggest for creating dynamic array of characters? Eg. while the game is running you get the information from socket event that new player has connected. Or even if I would like to add characters on some time interval?
Is making new instance with new config way to go?

**Comment by borkowsky** [comment, marked as answer]
Hi. I do the same (mmorpg game in browser). Look into addCharacter and removeCharacter methods.

**Reply by neu5** [reply]
@borkowsky thanks! How did I miss that? No idea :)

**Reply by borkowsky** [reply]
Haha, its not a problem :)

**Reply by splashsky** [reply]
> 
> 
> @borkowsky thanks! How did I miss that? No idea :)

Don't sweat it! I'm currently working on the documentation for the plugin, which I hope will make features like this much easier to pick up on.
